### PR TITLE
k8sfv: allow using any Felix version

### PR DESCRIPTION
**Pull request by neiljerram**

### CODE REVIEW

File: k8sfv/run-test [hunk 1]
```diff
@@ -97,20 +100,28 @@ while ! curl -k ${k8s_api_endpoint}/apis/extensions/v1beta1/thirdpartyresources;
...
 docker run --detach --privileged --name ${prefix}-felix \
+       -e CALICO_DATASTORE_TYPE=kubernetes \
        -e FELIX_LOGSEVERITYSCREEN=${FELIX_LOG_LEVEL} \
        -e FELIX_DATASTORETYPE=kubernetes \
        -e FELIX_PROMETHEUSMETRICSENABLED=true \
        -e FELIX_USAGEREPORTINGENABLED=false \
        -e FELIX_DEBUGMEMORYPROFILEPATH="heap-<timestamp>" \
        -e K8S_API_ENDPOINT=${k8s_api_endpoint} \
-       -e K8S_INSECURE_SKIP_TLS_VERIFY=true \
+       -e K8S_CA_FILE=/testcode/k8sfv/output/apiserver.crt \
```
Comment (fasaxc) [inline k8sfv/run-test]: Unrelated change?

Comment (neiljerram) [inline k8sfv/run-test]: Actually this is related, because Felix 2.1.x doesn't honour K8S_INSECURE_TLS_VERIFY. So to enable targeting 2.1.x we have to do certificate verification properly.

Comment (fasaxc) [inline k8sfv/run-test]: Ah.
